Geometry/VeryForwardGeometryBuilder/interface/DetGeomDesc.h

```diff
@@ -80,7 +80,7 @@ class DetGeomDesc {
   // params() is left for general access to solid shape parameters, but should be used
   // only with great care, for two reasons: 1. order of parameters may possibly change from
   // a version to another of DD4hep; 2. length parameters unit is cm while PPS uses mm.
```

Comment:
"// a version to another of DD4hep; 2. length parameters unit is cm while PPS uses mm." change to:
"// a version to another of DD4hep; 2. length parameters unit is the DD4hep configured unit while PPS uses mm."

Reply:
done